manager/integration/tests/test_basic.py

```diff
@@ -774,8 +776,178 @@ def test_backup_metadata_deletion():  # NOQA
+
+    v1bv, v1b1, _, _ = create_backup(client, volume1_name)
+    v2bv, v2b1, _, _ = create_backup(client, volume2_name)
+    _, v1b2, _, _ = create_backup(client, volume1_name)
+    _, v2b2, _, _ = create_backup(client, volume2_name)
+
+    bvs = client.list_backupVolume()
+
+    for bv in bvs:
+        backups = bv.backupList()
+        for b in backups:
+            assert b.messages is None
+
+    v1b1_new = v1bv.backupGet(name=v1b1.name)
+    assert v1b1_new.name == v1b1.name
+    assert v1b1_new.url == v1b1.url
+    assert v1b1_new.snapshotName == v1b1.snapshotName
+    assert v1b1_new.snapshotCreated == v1b1.snapshotCreated
+    assert v1b1_new.created == v1b1.created
+    assert v1b1_new.volumeName == v1b1.volumeName
+    assert v1b1_new.volumeSize == v1b1.volumeSize
+    assert v1b1_new.volumeCreated == v1b1.volumeCreated
+    assert v1b1_new.messages == v1b1.messages is None
+
+    v1b2_new = v1bv.backupGet(name=v1b2.name)
+    assert v1b2_new.name == v1b2.name
+    assert v1b2_new.url == v1b2.url
+    assert v1b2_new.snapshotName == v1b2.snapshotName
+    assert v1b2_new.snapshotCreated == v1b2.snapshotCreated
+    assert v1b2_new.created == v1b2.created
+    assert v1b2_new.volumeName == v1b2.volumeName
+    assert v1b2_new.volumeSize == v1b2.volumeSize
+    assert v1b2_new.volumeCreated == v1b2.volumeCreated
+    assert v1b2_new.messages == v1b2.messages is None
+
+    v2b1_new = v2bv.backupGet(name=v2b1.name)
+    assert v2b1_new.name == v2b1.name
+    assert v2b1_new.url == v2b1.url
+    assert v2b1_new.snapshotName == v2b1.snapshotName
+    assert v2b1_new.snapshotCreated == v2b1.snapshotCreated
+    assert v2b1_new.created == v2b1.created
+    assert v2b1_new.volumeName == v2b1.volumeName
+    assert v2b1_new.volumeSize == v2b1.volumeSize
+    assert v2b1_new.volumeCreated == v2b1.volumeCreated
+    assert v2b1_new.messages == v2b1.messages is None
+
+    v2b2_new = v2bv.backupGet(name=v2b2.name)
+    assert v2b2_new.name == v2b2.name
+    assert v2b2_new.url == v2b2.url
+    assert v2b2_new.snapshotName == v2b2.snapshotName
+    assert v2b2_new.snapshotCreated == v2b2.snapshotCreated
+    assert v2b2_new.created == v2b2.created
+    assert v2b2_new.volumeName == v2b2.volumeName
+    assert v2b2_new.volumeSize == v2b2.volumeSize
+    assert v2b2_new.volumeCreated == v2b2.volumeCreated
+    assert v2b2_new.messages == v2b2.messages is None
+
+    backupstore_overwrite_backup_cfg_file(client, core_api,
+                                          volume1_name, v1b1.name)
+
+    bvs = client.list_backupVolume()
+
+    for bv in bvs:
+        if bv.name == volume1_name:
+            backups = bv.backupList()
+            for b in backups:
+                if b.name == v1b1.name:
+                    assert b.messages is not None
+                else:
+                    assert b.messages is None
+
+    v1b2_new = v1bv.backupGet(name=v1b2.name)
+    assert v1b2_new.name == v1b2.name
```

Comment:
can use to helper mentioned above to make the code more readable

Reply:
fixed